src/shared/components/curriculum-box/curriculum-box.tsx

```
@@ -0,0 +1,17 @@
+import { glassEffect } from '@/shared/styles/glass.css';
+import * as styles from './curriculum-box.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)